tests/fixed/UselessConditions.php

```diff
@@ -44,12 +44,20 @@ public function unecessaryIfMethodForEarlyReturn() : bool
 
     public function uselessIfConditionWithParameter(bool $bool) : bool
     {
-        return ! $bool;
```

Comment:
See https://github.com/slevomat/coding-standard/issues/604 - CS doesn't know the type of `$bool` and in general it may produce broken code (the method would return non-bool). This is now still reported, but not auto-fixed.

Reply:
Instead of changing the test to pass with the new impeachmentation, would it make more sense to remove it? The updated test doesn’t describe how we want the code to work.

Reply:
It's still reported as error, but the error is no longer fixable.